pycqed/analysis_v2/base_analysis.py

```diff
@@ -199,9 +204,39 @@ def process_data(self):
     def prepare_plots(self):
         pass
 
-    def run_fitting(self):
+    def prepare_fitting(self):
```

Comment:
Adding a docstring here would be very helpful. (Similar to what there is on my branch)